rules/windows/lateral_movement_scheduled_task_powershell_source.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/12/15"
+maturity = "production"
+updated_date = "2020/12/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies powershell process loading the Task Scheduler COM DLL followed by an outbound RPC network connection within a
+short time period. This may indicate lateral movement or remote discovery via scheduled tasks.
+"""
+false_positives = ["Legitimate scheduled tasks may be created during installation of new software."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Outbound Scheduled Tasks Activity via PowerShell"
+references = [
+    "https://www.volexity.com/blog/2020/12/14/dark-halo-leverages-solarwinds-compromise-to-breach-organizations/",
+]
+risk_score = 43
+rule_id = "5cd55388-a19c-47c7-8ec4-f41656c2fded"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery", "Lateral Movement"]
+type = "eql"
+
+query = '''
+sequence by host.id, process.entity_id with maxspan = 5s
+ [library where file.name: "taskschd.dll" and process.name: ("powershell.exe", "pwsh.exe")]
+ [network where process.name : "powershell.exe" and destination.port == 135 and not destination.address in ("127.0.0.1", "::1")]
```

Comment:
For the network connection, does the `pwsh.exe` process also need be accounted for too?

Reply:
nice catch :)